indico/modules/events/management/templates/assign_program_codes.html

```diff
@@ -27,6 +27,13 @@
                 </div>
             {% endif %}
             {{ object.full_title or object.title }}
+            {%- if object_type in ('contributions', 'subcontributions')-%}
```

Comment:
might need `and object.speakers` depending on whether an empty list looks strange there or not (it'd be cleaner in any case)